spec/services/benefit_check_service_spec.rb

```diff
@@ -56,7 +56,7 @@
       end
     end
 
-    context 'when the API raises an error', vcr: { cassette_name: 'benefit_check_service/service_error' } do
+    context 'calling API raises a Savon::SOAPFault error', vcr: { cassette_name: 'benefit_check_service/service_error' } do
```

Comment:
Curious how the `expect(subject.call).to eq false` in this context was passing before 🤔

Reply:
Great Spot! I'm trying to find out now.

Reply:
In the test / dev environment, we don't send to Sentry then Raven returns false. In production we send to sentry and so Raven returns its raven event object which lead to the sentry issue.